bridge/client/app/_models/event-icons.ts

```diff
@@ -1,7 +1,7 @@
 export const EVENT_ICONS = {
   "artifact-delivery": "duplicate",
   "deployment": "deploy",
-  "test": "perfromance-health",
+  "test": "performance-health",
```

Comment:
Unfortunately that's a typo in another library :D I'll revert this back after the merge, don't worry.
https://barista.dynatrace.com/resources/icons/perfromance-health

Reply:
@lukasholzer the only places I can find this token are in Dynatrace/sketchmine: https://github.com/Dynatrace/sketchmine/search?q=perfromance-health
but that repository is archived.